extras/rxjava/src/main/java/org/asynchttpclient/extras/rxjava/AsyncHttpClientErrorException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015 Target, Inc. All rights reserved.
```

Comment:
Sorry, we can't accept PRs where code isn't donated to "The AsyncHttpClient Project".